bpf/process/bpf_execve_bprm_commit_creds.c

```diff
@@ -70,6 +66,16 @@ BPF_KPROBE(tg_kp_bprm_committing_creds, struct linux_binprm *bprm)
 
 	memset(&heap->info, 0, sizeof(struct execve_info));
 
+	/* Read binary file information */
+	probe_read(&file, sizeof(file), _(&bprm->file));
```

Comment:
Why not use something like `file = BPF_CORE_READ(bprm, file);` to be consistent with the next lines?

Reply:
Alright, I converted it and added some checks to ensure if it did not fail, but I don't propagate errors back into maps.

I will do those in a follow PR as I have to check how we are doing this since your last update.